sycl/include/sycl/string.hpp

```diff
@@ -0,0 +1,42 @@
+//==----------------- string.hpp - SYCL standard header file ---------------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+#include <iostream>
+
+#pragma once
+
+namespace sycl {
+inline namespace _V1 {
+
+class string {
+  const char *str; // used to send existing std::string to libsycl
+  char *ret_str;   // set from libsycl
```

Comment:
```suggestion
  char *ret_str = nullptr;   // set from libsycl
```
Otherwise, it's left uninitialized by two constructors.

Reply:
Done. Thanks.